ocaml/libs/tracing/tracing.ml

```diff
@@ -138,20 +166,20 @@ module Span = struct
             )
         in
         let status_code = Status.Error in
-        let exn_tags =
+        let exn_attributes =
           [
             ("exception.message", msg)
           ; ("exception.stacktrace", stacktrace)
           ; ("exception.type", exn_type)
+          ; ("error", "true")
           ]
         in
         match span.status.status_code with
         | Unset ->
-            {
-              span with
-              status= {status_code; description}
-            ; tags= span.tags @ exn_tags
-            }
+            let attributes =
+              Attributes.union (fun _k a _b) span.attributes (attrs_of_list exn_attributes)
```

Comment:
Sorry, I put an incomplete function, it should have been (fun _k a _b -> Some a), like above